scripts/globals/items/teleport_ring_yhoat.lua

```diff
@@ -1 +1 @@
------------------------------------------
--    ID: 14665
--    Teleport ring: Yhoat
--  Enchantment: "Teleport-Yhoat"
-----------------------------------------

require("scripts/globals/status");
require("scripts/globals/keyitems");
require("scripts/globals/teleports");

-----------------------------------------
-- OnItemCheck
-----------------------------------------

function onItemCheck(target)
    local result = 0;
    if (target:hasKeyItem(YHOATOR_GATE_CRYSTAL) == false) then
        result = 445;
    end
    return result;
end;

-----------------------------------------
-- OnItemUse
-----------------------------------------

function onItemUse(target)
    target:addStatusEffectEx(EFFECT_TELEPORT,0,TELEPORT_YHOAT,0,0.5);
end;
\ No newline at end of file
+-----------------------------------------
--    ID: 14665
--    Teleport ring: Yhoat
--  Enchantment: "Teleport-Yhoat"
-----------------------------------------

require("scripts/globals/status");
require("scripts/globals/keyitems");
require("scripts/globals/teleports");

-----------------------------------------
-- OnItemCheck
-----------------------------------------

function onItemCheck(target)
    local result = 0;
    if (target:hasKeyItem(YHOATOR_GATE_CRYSTAL) == false) then
        result = 445;
    end
    return result;
end;

-----------------------------------------
-- OnItemUse
-----------------------------------------

function onItemUse(target)
    target:addStatusEffectEx(EFFECT_TELEPORT,0,TELEPORT_YHOAT,0,1);
end;
```

Comment:
Nah, that's not encoding, that's a windows line break going to a linux line break. I'm working off of a debian fork server and it doesn't care for windows files. I can fix that pretty quickly.